admin/acceptance.py

```diff
@@ -156,23 +127,26 @@ def __init__(self):
             raise UsageError("Unsupored varianta: %s."
                              % (', '.join(self.variants),))
 
-    def start_nodes(self):
+    @inlineCallbacks
```

Comment:
That might be a valid enough reason but I don't want to make that call. Perhaps @exarkun or @itamarst would like to chime in?